docs/07.DEBUGGER.md

```diff
@@ -194,3 +201,33 @@ jerry_debugger_stop_at_breakpoint (bool enable_stop_at_breakpoint)
   jerry_debugger_stop_at_breakpoint (false);
 }
 ```
+
+### jerry_debugger_wait_and_run_client_source
+
+**Summary**
+
+Stops the engine and puts that into a waiting loop. If the client send
+a source code and the JerryScript receive that, then the function will
+run the source with the initialized options.
+
+**Prototype**
+
+```c
+jerry_debugger_wait_and_run_type_t
+jerry_debugger_wait_and_run_client_source (jerry_value_t *return_value)
+```
+
+**Example**
+
+```c
+  jerry_init (JERRY_INIT_DEBUGGER);
+
+  jerry_value_t wait_and_run_value = jerry_create_undefined ();
```

Comment:
No value should be passed to the function. In fact it does not even release the value as far as I remember.